src/components/Practice/Mechanics7.jsx

```
@@ -509,94 +509,130 @@ const Mechanics7 = ({
+                      border: getBorder(),
+                      color: colors,
+                      fontWeight: type === "word" ? 600 : 700,
                       fontSize: "22px",
                       fontFamily: "Quicksand",
-                      marginLeft: "5px",
-                      role: "button",
+                      cursor: "pointer",
+                      marginLeft:
+                        type === "word" ? 0 : answer != "correct" ? "20px" : 0,
                     }}
                   >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)